waffle-chai/test/events.ts

```diff
@@ -5,11 +5,14 @@ import {EVENTS_ABI, EVENTS_BYTECODE} from './contracts/Events';
 
 describe('INTEGRATION: Events', () => {
   const [wallet] = new MockProvider().getWallets();
+  let eventFactory: ContractFactory;
   let events: Contract;
 
   beforeEach(async () => {
     const factory = new ContractFactory(EVENTS_ABI, EVENTS_BYTECODE, wallet);
     events = await factory.deploy();
+    eventFactory = new ContractFactory(EVENTS_ABI, EVENTS_BYTECODE, wallet);
```

Comment:
There is no need to have two identical factories. Store the one from line 12 in a variable and then use it in the new test